src/test/java/training/AppTest.java

```diff
@@ -1,14 +1,66 @@
-/*
- * This Java source file was generated by the Gradle 'init' task.
- */
-package training;
-
-import org.junit.Test;
-import static org.junit.Assert.*;
-
-public class AppTest {
-    @Test public void testAppHasAGreeting() {
-        App classUnderTest = new App();
-        assertNotNull("app should have a greeting", classUnderTest.getGreeting());
-    }
+  
+  package playground;
+
+class Shapes
```

Comment:
Like earlier the App class should be used . 

Also why this whole shapes code is in the tst directory

Tst directory is for writing tests.